content/api/alerts.apib

```diff
@@ -1,205 +1,322 @@
+label: New
 
 # Group Alerts
-Alerts can be used to get notifications regarding various activities on your SparkPost account.
+Alerts can be used to get notifications regarding various activities and metrics on your SparkPost account.
 
-Alerts can be configured to send notifications via email to up to 10 email addresses, and supports the following activity:
-* Approaching monthly sending limit
+## Metrics
+At this time, the following metrics are supported:
 
-Alerts are configured to be account-wide. Currently, subaccount level alerts are not supported. 
+* Real-Time
+    + monthly_sending_limit
+    + block_bounce_rate
+* Signals
+    + health_score
+
+Each metric group has a specific set of filters and some other values allowed.
+
+* Real-Time
+    + filters - filter_type can be 'sending_domain', 'sending_ip', or 'mailbox_provider'
+    + threshold_evaluator - source must be 'raw'
+* Signals
+    + filters - filter_type can be 'sending_domain', 'ip_pool', or 'mailbox_provider'. Only one filter type can be selected,
+        though multiple values are allowed.
+
+* monthly_sending_limit
+  + filters - must be an empty array. No filters may be applied to this metric.
+  + subaccount - must be 0.
+  + any_subaccount - must be unset or set to false.
+
+## Channels
+Alerts can have a number of different notification channel types associated with them. At this time, the following channels are supported:
+
+* emails - Sends notifications via email to up to 10 email addresses.
+
+## Filter Object
+
++ Data Structure: Attributes
+  + filter_type (enum) - Type of filter to apply to the alert.
+    + `"sending_domain", "mailbox_provider", "sending_ip", "ip_pool"`
+  + `filter_values` (array) - Set of all filter values associated with the given filter_type.
+
++ Sample
+
+  ```
+  {
+    "filter_type": "sending_domain",
+    "filter_values": [
+      "sparkpost.com",
+      "test.com"
+    ]
+  }
+  ```
 
 ## Alert Object
 
 + Data Structure: Attributes
+    + id (number) - Unique identifier of the alert.
     + name (string) - Display name of the alert.
-    + description (string) - Description of the alert.
-    + id (string) - Unique alphanumeric id.
-    + email_addresses (array) - List of email addresses to send alert notifications.
-    + alert_metric (enum) - Metric that the alert is configured to track.
-        + `"monthly_sending_limit"`
-    + threshold (object) - Configuration for level and threshold at which an alert is generated.
-        + error (object) - Error level alert configuration.
-            + target (number) - Percentage value at which an alert will be generated.
-            + comparator (enum) - Conditional comparison operator used by alerts service to compare actual value of `alert_metric` to the `target` value.
-                + `"gt"`
-    + enabled (boolean) - Whether or not the alert is active.
+    + subaccount (number) - Subaccount filter applied to the alert.
```

Comment:
Is subaccount different than a regular filter?

Reply:
Yes, in that both subaccount and up to 1 filter may be set for Signals alerts.